### DISCUSSION TITLE
Nuxt Generate: Not creating subFolders

### DISCUSSION THREAD

**Discussion by iandk** [nuxt/nuxt#17668] [Questions]
Hi, 

I had to notice that nuxt generate doesn't seem to create subfolders for the pages anymore. 
NuxtJS v2 created a separate folder for each site with an index.html in it, NuxtJS v3 only creates a single .html file for each site.

This is is a problem if you want to host you site on a static hosting provider without fallback support.
Therefore each page gets a separate folder with an index.html file in it. 

I couldn't find anything in the v3 docs regarding this, at least in v2 it seemed to be turned on by default.
So I'm not sure if this is a bug or I'm doing something wrong.

How can I enable this option for NuxtJS 3?

NuxtJS 2 docs:
https://nuxtjs.org/docs/directory-structure/dist/#the-subfolders-property

**Comment by pi0** [comment]
Thanks for pointing this out! I've created an upstream issue to track: https://github.com/unjs/nitro/issues/166
